renovate.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "driver_config.yml"
+      ],
+      "matchStrings": [
+        "cuda:\\s*(?:#.*)?\\n\\s*version:\\s*\"(?<currentValue>[^\"]+)\""
+      ],
+      "depNameTemplate": "nvidia-cuda-driver",
+      "datasourceTemplate": "custom.nvidia-driver",
+      "versioningTemplate": "loose"
+    }
+  ],
+  "customDatasources": {
+  "nvidia-driver": {
+    "defaultRegistryUrlTemplate": "https://docs.nvidia.com/datacenter/tesla/drivers/releases.json",
+    "transformTemplates": [
+      "{ \"releases\": $map($reduce($map($keys($), function($k) { { \"key\": $k, \"value\": $lookup($, $k) } }), function($acc, $v) { $append($acc, $v.value.driver_info) }, []), function($info) { { \"version\": $info.release_version } }) }"
+    ]
+  }
+ },
+  "packageRules": [
```

Comment:
what is this packageRule for ?

Reply:
Had it to see about reducing the files it was tracking - unnecessary now